### PR TITLE
Force re-draw shadow when bounds change

### PR DESCRIPTION
**Pull request by pedrodanielcsantos**

### CODE REVIEW

File: MaterialLibrary/src/main/java/io/doist/material/elevation/CompatElevationDrawable.java [hunk 1]
```diff
@@ -345,20 +345,26 @@ private void update(boolean force, boolean async) {
             int bottom = top + mBounds.height();
 
             if (force || left != mLeft || top != mTop || right != mRight || bottom != mBottom) {
+                // Check if there were changes in the view's dimensions.
```
Comment (goncalossilva) [inline MaterialLibrary/src/main/java/io/doist/material/elevation/CompatElevationDrawable.java]: Nit: _drawable's dimensions_?

File: MaterialLibrary/src/main/java/io/doist/material/elevation/CompatElevationDrawable.java [hunk 2]
```diff
@@ -345,20 +345,26 @@ private void update(boolean force, boolean async) {
...
                 int shadowLengthBottom = getShadowLengthBottom(bottom);
 
-                boolean leftDirty = mShowShadowLeft && (force || shadowLengthLeft != mShadowLengthLeft);
-                boolean topDirty = mShowShadowTop && (force || shadowLengthTop != mShadowLengthTop);
-                boolean rightDirty = mShowShadowRight && (force || shadowLengthRight != mShadowLengthRight);
-                boolean bottomDirty = mShowShadowBottom && (force || shadowLengthBottom != mShadowLengthBottom);
+                boolean leftDirty = mShowShadowLeft && (force || shadowLengthLeft != mShadowLengthLeft || widthChanged);
+                boolean topDirty = mShowShadowTop && (force || shadowLengthTop != mShadowLengthTop || heightChanged);
+                boolean rightDirty =
+                        mShowShadowRight &&(force || shadowLengthRight != mShadowLengthRight || widthChanged);
```
Comment (goncalossilva) [inline MaterialLibrary/src/main/java/io/doist/material/elevation/CompatElevationDrawable.java]: Missing whitespace after `&&`.

**Review by goncalossilva — APPROVED** [review]
Great catch and fix! 😊 Code looks good as well.

**Comment by pedrodanielcsantos** [comment]
Thanks a lot fr the quick review @goncalossilva ! Amended the commit with the fix. 👍

**Pull request merged**